include/boost/winasio/winhttp/winhttp_asio.hpp

```diff
@@ -55,6 +55,9 @@ template <typename Executor> class asio_request_context {
   // need to invoke query data inside
   net::windows::overlapped_ptr on_read_complete;
 
+  // need to invoke send http request failure
+  net::windows::overlapped_ptr on_send_request_failed;
```

Comment:
What function will this invoke? where is this initialized. I think currently it is empty.

Reply:
I thought the complete function of the overlapped pointer is enough to finish the callback. Do we need a separate callback function for error handling? I saw all other cases just used complete().

Reply:
You added an empty completion ptr, where it is no-op. So your code does not do anything.